e2e-polybft/e2e/nexus_bridge_test.go

```diff
@@ -1939,6 +1939,131 @@ func TestE2E_ApexBridgeWithNexus_BatchFailed(t *testing.T) {
 		sendAmountDfm, sendAmountEth = convertToEthValues(1)
 	}
 
+	t.Run("Test insufficient gas price dynamicTx=true", func(t *testing.T) {
```

Comment:
Just a thought: maybe you should put
```
if shouldSkip := os.Getenv("SKIP_E2E_REDUNDANT_TESTS"); shouldSkip == "true" {
			t.Skip()
		}
```
on top (`func TestE2E_ApexBridgeWithNexus_BatchFailed(t *testing.T)`) because all these test should be skipped?